source/manual/github.html.md

```diff
@@ -34,46 +34,45 @@ There are several GOV.UK GitHub teams within alphagov, including:
+If your role requires access to Github, you'll need to be added to the organisation and the relevant team(s) through Terraform, in [govuk-user-reviewer][govuk-user-reviewer] - _not_ manually added through the GitHub UI itself, as this breaks the Terraform setup. Note that you will have to accept the invite sent via email before you're added to the organisation.
 
-- If you're a content designer, ask for GitHub access via Zendesk (see [example ticket](https://govuk.zendesk.com/agent/tickets/5812930/events)). Ensure to include `govuk_platform_support` tag.
-- If you're an engineer or contractor, ask your tech lead to follow the instructions in [govuk-user-reviewer][govuk-user-reviewer] to add you.
-- If you don't have a tech lead, [ask someone in Senior Tech](/manual/ask-for-help.html#contact-senior-tech) to add you. You must state
-  - your role
-  - which team you're in
-  - your GitHub handle
-  - which GitHub team(s) you should join ([see list](#govuk-teams))
-  - why you need access
+- If you're an engineer or contractor, ask your tech lead to add you by following the instructions in [govuk-user-reviewer][govuk-user-reviewer].
+- If you don't have a tech lead, [ask someone in Senior Tech](/manual/ask-for-help.html#contact-senior-tech) to add you.
+- If you're a content designer, ask for GitHub access via Zendesk (see [example ticket](https://govuk.zendesk.com/agent/tickets/5812930)). Make sure you include the `govuk_platform_support` tag.
 
-## Removing access to GitHub
-
-Users are removed from the GitHub organisation when their entry in [govuk-user-reviewer][govuk-user-reviewer] is deleted.
+In these last two cases, you must state:
 
-# GOV.UK repos
+- your role
+- which team you're in
+- why you need access
+- your GitHub username
+- which GitHub team(s) you should join ([see list](#govuk-teams))
 
-## Create and configure a new GOV.UK repo
+## Removing access to GitHub
 
-When creating a new GOV.UK repo, you must ensure it:
+Users are removed from the GitHub organisation when their entry in [govuk-user-reviewer][govuk-user-reviewer] is deleted.
 
-- has a well-written README (see [READMEs for GOV.UK applications](/manual/readmes.html), or the [GDS Way guidance](https://gds-way.digital.cabinet-office.gov.uk/manuals/readme-guidance.html#writing-readmes) for general repositories)
-- is tagged with the [`govuk`](https://github.com/search?q=topic:govuk) topic
-- has [Dependency Review](/manual/dependency-review.html) and [CodeQL](/manual/codeql.html) scans in its CI pipeline
-- is added to the [repos.yml](https://github.com/alphagov/govuk-developer-docs/blob/main/data/repos.yml) file in the Developer Docs.
+## Creating and configuring a new GOV.UK repo
```

Comment:
I would almost make it a new separate page, it doesn't seem to come up in search otherwise.

Reply:
Thanks, yes I wondered that. Will have a go